packages/blockchain-wallet-v4-frontend/src/components/QRCodeCapture/index.js

```diff
@@ -8,6 +8,8 @@ import bip21 from 'bip21'
 
 import { actions } from 'data'
 import QRCodeCapture from './template.js'
+import * as C from 'services/AlertService'
+import {utils} from '../../../../blockchain-wallet-v4/src'
```

Comment:
lets not use relative paths. should be `import {utils} from 'blockchain-wallet-v4/src'`

Reply:
done